lib/diagnostic.js

```diff
@@ -7,38 +7,56 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min () {
-
+const min = function min (num1, num2) {
+  return num1 - num2
```

Comment:
Yup, but you want this function to be able to take an unlimited number of arguments. You can use the `arguments` object inside your function